core/profile_test.go

```diff
@@ -73,6 +76,105 @@ func TestProfileRequestsSave(t *testing.T) {
 	}
 }
 
+func TestSaveProfile(t *testing.T) {
+	// Mock data for the global Config's Profile, used to create new profile.
+	// TODO: Remove the randomly built Profile that config.DefaultProfile creates.
```

Comment:
I like this TODO. It'd be great if we put it in `profile.NewRandomProfile()` or something